.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Thanks for contributing to the `fabric-chaincode-evm` project!
+
+Please make sure you have done the following before submitting:
+
+* [ ] Ran the basic checks: `make basic-checks`
+
+* [ ] Ran the unit tests: `make unit-test`
+
+* [ ] Ran the integration tests: `make integration-test`
+
+* [ ] Added the associated JIRA ticket number in the commit message title
+
+* [ ] Link to the JIRA Ticket:
+
+If you do not have a corresponding JIRA ticket, please open one in the [Fabric
+JIRA](https://jira.hyperledger.org/projects/FAB/issues) and add
+`fabric-chaincode-evm` as the component. In the JIRA ticket, include use
+cases, design and other information useful to understanding why you are making
+this pull request.
```

Comment:
We're gonna get so many PRs with no JIRA.

Reply:
If it is in the PR template, those who are new have the information to make a contribution.